portfolio-alex.css

```diff
@@ -0,0 +1,237 @@
+*{
+	box-sizing:border-box;
+}
+
+
+body {
+	margin: 0;
+	padding: 0;
+	font-size: 62.5%;
+}
+
+.topheader {
+	width: 100%;
+	top: 0;
+}
+	
+.logo {
+	background-color: #008080;
+	width: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	text-align: center;
+	border-bottom: solid;
+}
+
+.logotext {
+	font-size: 1.8rem;
+	font-family: "spartan";
+	width: 60%;
+	margin: center;
+}
+@media only screen and (max-width: 900px) {
+  .logotext {
+	  font-size: 1rem;
+  }
+}
+
+.navlinks {
+	border: none;
+	font-size: 1.5rem;
+	padding: 10px;
+	line-height: 50px;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	height: 1px;
```

Comment:
this isnt a good solution. instead use position relative on the parent, and position absolute with bottom -20px (or something) to move the nav